rollup.config.js

```diff
@@ -1,53 +1,75 @@
-import resolve from 'rollup-plugin-node-resolve';
```

Comment:
it looks like all single quotes were changed to double quotes.  I don't have a strong preference here but other repos in this org seem to use single quotes in rollup config.

Reply:
Changed to single quotes to abide by project